lib/_http_server.js

```diff
@@ -1063,6 +1069,10 @@ function parserOnIncoming(server, socket, state, req, keepAlive) {
     server.emit('request', req, res);
   }
 
+  if (onRequestEndChannel.hasSubscribers) {
+    onRequestEndChannel.publish(dcMessage);
```

Comment:
I guess a try block is needed to ensure end is called even if an exception is thrown.

Reply:
Yep. And also should have the error channel to fully match tracing channel.